Tasks/AppCenterDistribute/task.json

```diff
@@ -12,7 +12,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 131,
+        "Minor": 134,
```

Comment:
*If* we want to rename `distributionGroupId` to `destinationId`. Will this be good enough to avoid breaking current customers? Do we need to bump the major version?

Reply:
Renaming input names is a breaking change and will need a major version increment

Reply:
If you want to rename it just for yaml without a breaking change, you can add `"aliases": [ "destinationId" ]` to task.json, allowing people to use either input name in yaml (and the API?). Build definitions saved in the designer will still use `distributionGroupId` though.

If you want to fully change the name, a breaking change is necessary.

Reply:
Talked to Garrett (Distribution PM). Let's make the change and bump the major version

Reply:
Done, @DennisPan